enterprise/poplar-hoperun/additional-docs/README.md

```diff
@@ -1,18 +1,21 @@
 ---
-title: Additional Docs for boardname
-permalink: /documentation/consumer/poplar-hoperun/additional-docs/
+title: Additional Documentation for Hoperun Poplar
+permalink: /documentation/enterprise/poplar-hoperun/additional-docs/
+redirect_from:
+- /documentation/EnterpriseEdition/Poplar-Hoperun/AdditionalDocs/README.md/
+- /documentation/EnterpriseEdition/Poplar-Hoperun/AdditionalDocs/
+- /documentation/enterprise/poplar-hoperun/additional-docs/README.md
```

Comment:
since this is a new board there is no need for the redirect_from tag as it can cause errors with the link checker.
please remove from all .md files where this exists

Reply:
@ric96 Just fix it according to your suggestion, many thanks.